routeid/routeid.go

```diff
@@ -0,0 +1,29 @@
+package routeid
```

Comment:
which test? what I mean in case option one, is that one `if` statement and random string generation with limited set of characters can be copypasted, yes, and doesn't even need to be separately tested. what needs to be tested is the calling code, that whatever it does, the route gets an id.

for the second option, the eskip package is fine, because that's where the route type and the id field is defined.

No, I am not afraid of creating new packages, as you can see by the packages I already created. See what I said above: please don't create a package with a single function. To be more precise: please don't create a new package with a single function that is as trivial as this.